src/v2/Apps/Components/AppShell.tsx

```diff
@@ -120,6 +124,18 @@ export const AppShell: React.FC<AppShellProps> = props => {
 
       <MNTNConversionPixel />
       <MNTNTrackingPixel />
+
+      {isMounted && <OneTrustModalOverlayHotfixStyles />}
     </Flex>
   )
 }
+
+/**
+ * This is a workaround for the cookie consent banner overlay appearing on top
+ * of our modals. This positions it below so that modal buttons are clickable.
+ */
+export const OneTrustModalOverlayHotfixStyles = createGlobalStyle`
+  #onetrust-banner-sdk {
+    z-index: 1 !important;
+  }
+`
```

Comment:
Why wouldn't you do this like the rest of the styles? My concern would be now there's styling in two places. There should be a FIXME here to just move this onto the inlined styles on OneTrust.